src/app/app.module.ts

```diff
@@ -1,18 +1,17 @@
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
+import { ReactiveFormsModule } from '@angular/forms';
+import { FormsModule } from '@angular/forms'; // <-- import this module
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { CreateButtonComponent } from './components/create-button/create-button.component';
+import { AddCardComponent } from './components/add-card/add-card.component';
 
 @NgModule({
-  declarations: [
-    AppComponent
-  ],
-  imports: [
-    BrowserModule,
-    AppRoutingModule
-  ],
+  declarations: [AppComponent, CreateButtonComponent, AddCardComponent],
```

Comment:
Organizing code this way will make it more readable.

```suggestion
  declarations: [
    AppComponent, 
    CreateButtonComponent,
    AddCardComponent
  ],
```

Reply:
I am using Prettier Code Formatter. Its default configuration automatically formats in this way. if we have more than three components in declarations then it organizes it in your suggested way